### DISCUSSION TITLE
Expand "Side-Panel / Preview" Functionality to PRs as well in Project View

### DISCUSSION THREAD

**Discussion by MrChocolateMoose** [community/community#59684] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

In Github Projects, you can see a side-panel preview of issues from the various project views, but PRs opens in a new window. 

The request is to be able to:
- Open up a PR type item in the same side-panel fashion
- Clicking "Linked Pull Requests" would open it it in a side-panel fashion
- Clicking a linked pull request in a side-panel issue would open the PR in a side-panel 

This would help optimize a lot of workflows, especially when running team meetings.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Kiryous** [comment]
+1. Daily-driver setup: split view with the project board on the left and PR detail on the right. Gmail-style. Issue side panel works; PRs opening in a new tab breaks it.
